src/index.js

```diff
@@ -28,6 +28,7 @@ import {
   ColdcardSignMultisigTransaction,
   ColdcardMultisigWalletConfig,
 } from './coldcard';
+import {COBOVAULT, CoboVaultSignMultisigTransaction} from "./cobovault";
```

Comment:
Will want to add support here for `CoboVaultExportExtendedPublicKey`, no? Right now only exposing `CoboVaultSignMultisigTransaction` via this index file, though it looks like extendedpublickey is implemented in `cobovault.js`